docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:buster-slim
+
+RUN set -ex \
+    && apt-get update \
+    && apt-get install -y \
+	build-essential \
+	python2.7-dev \
+	libffi-dev \
+	sqlite3 \
+	python-pip \
+	libssl-dev \
+	python-virtualenv \
+	libxslt1-dev
+
+# Install dependencies
+RUN pip install https://github.com/matrix-org/sydent/tarball/master \
```

Comment:
You probably want to run this last in the run script.

pip and setuptools can also break when you go down this path as I found while trying to build this out myself [here](https://github.com/drGrove/docker-sydent/blob/master/Dockerfile)

email templates wont translate over from a `pip install`.

Running as root is also incredibly dangerous as you can break out of the container